keyboard/redir.php

```diff
@@ -13,7 +13,7 @@
     $v = "$q/{$_REQUEST['v']}";
     if(!file_exists("$v/$q.php")) {
       unset($v);
-    }
+  }
```

Comment:
Maybe undo this removal of indentation?

```suggestion
    }
```